packages/hash/external-services/kratos/kratos.yml

```diff
@@ -0,0 +1,61 @@
+# Based on the quick-start example https://github.com/ory/kratos/blob/master/contrib/quickstart/kratos/email-password/kratos.yml
```

Comment:
(comment for file)
Would we want a dev and non-dev version of the kratos config?
For Alfie's comment further down this file, it would make sense for the dev environment to be verbose and potentially leak sensitive information, but not for prod. Thoughts on that?

Reply:
Yes we should have a production kratos config, this is not that configuration though. I assume that configuration will not be made within `external-services` which uses docker-compose, and more likely live in a terraform configuration file in a `deployment` folder or something.

I think we should improve the naming of these packages to make it more clear what is intended for dev purposes, and what is intended for a production environment. But I'm not sure we should try and do that as part of the `dev/graph` branch.